cumulusci/robotframework/explicit_failures/failures.robot

```diff
@@ -0,0 +1,15 @@
+*** Settings ***
```

Comment:
To me, the test name is clear that if you want to test what happens (in any infrastructure) when a Robot test fails then you WOULD run this test and if you don't want to, then you would NOT run the test. Generally failing tests break our builds so by default you wouldn't want to run it. But if you wanted to test some aspect of the infrastructure WITH a failing test then you WOULD run it.

It's easy to move it, but I don't really understand what is confusing about how it is named or what it does. It uses the word "failure" twice in its path to say that it is a set of failing tests.

Reply:
I think it belongs alongside the pytest file that uses it. For example, **cumulusci/tasks/robotframework/tests/failures.robot** or **failing_tests.robot** or something like that, since it's used by **test_robotframework.py** in that folder. We already have a precedent where tests in **test_robotframework.py** use a .robot file in that same folder. 

If you do that, you should be able to replace this:

`suite = Path(self.datadir) / f"../../../robotframework/{suite_path}"
`

with a much simpler:

`suite = Path(self.datadir) / "failures.robot"        
`

At some point, I'll probably create a **test_data** folder for assets like this so it's more clear that the files are used by other tests, but I don't expect you to do that now.

Reply:
Okay, moved.